src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt

```diff
@@ -4,20 +4,39 @@ import com.atlassian.performance.tools.jiraactions.api.page.wait
 import com.atlassian.performance.tools.jirasoftwareactions.webdriver.JavaScriptUtils
 import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
 import org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated
+import org.openqa.selenium.support.ui.ExpectedConditions.textToBePresentInElementLocated
 import java.time.Duration
 
 class ViewBacklogPage(
     private val driver: WebDriver
 ) {
-    fun waitForBacklog() = driver.wait(
-        Duration.ofSeconds(30),
-        presenceOfElementLocated(By.cssSelector("#ghx-backlog[data-rendered]:not(.browser-metrics-stale)"))
-    )
+    private val sprintsLocator = By.cssSelector(".ghx-sprint-group .ghx-backlog-container")
+    private val duration = Duration.ofSeconds(30)
```

Comment:
Nit: This deduplicated duration here is used in 2 different contexts, however both of those contexts can have different durations. Maybe we don't want to deduplicate it?

Reply:
Similar duplication can be found in `SprintOnBacklog`.

Reply:
@ewefie Has it been tested with JPT?

Reply:
This lib has its own integration test. The new action is covered, because it's included in the `Scenario`

Reply:
Well, in `ViewBacklogPage` I did this deduplication because I believe the complexity of both actions is similar and might require the same amount of time to successfully finish. But I see your point of view and I think I'll revert this change. However, in `SprintOnBacklog` I think that it makes sense to have one timeout used in those two conditions, both of them are closely related and consist of the same type of `ExpectedContidion`.

Reply:
@mgrzaslewicz during the development I was using `./gradlew testVirtualUserActions` to verify if the action works correctly.